tools/squidclient/squidclient.cc

```diff
@@ -424,20 +424,14 @@ main(int argc, char *argv[])
         atexit(Win32SockCleanup);
     }
 #endif
+
     /* Build the HTTP request */
     if (strncmp(url, "mgr:", 4) == 0) {
         char *t = xstrdup(url + 4);
-        const char *at = nullptr;
-        if (!strrchr(t, '@')) { // ignore any -w password if @ is explicit already.
-            at = ProxyAuthorization.password;
-        }
-        // embed the -w proxy password into old-style cachemgr URLs
-        if (at)
-            snprintf(url, sizeof(url), "cache_object://%s/%s@%s", Transport::Config.hostname, t, at);
-        else
-            snprintf(url, sizeof(url), "cache_object://%s/%s", Transport::Config.hostname, t);
+        snprintf(url, sizeof(url), "/squid-internal-mgr/%s", t);
```

Comment:
As discussed in another change request, `url` should probably be absolute. If you make it absolute, please do not forget to add Transport::Config.port! The official code kind of ignored the custom port issue but "usually worked OK", probably because the default `cache_object` scheme port is the default cache manager port. Unfortunately for us, the default `http(s)` scheme port usually is not the default cache manager port!

To keep things simple, I would not worry about sending an explicit `:80` or `:443` port in `squid-internal-mgr` URLs when squidclient is configured to talk to a proxy running at that port. It is excessive, but probably not worth the extra code.

Should use `https` scheme when `Config.tlsEnabled`? These `mgr:command` requests are always targeting the proxy squidclient connects to. Thus, that proxy should not forward these generated URLs to another proxy and will probably be OK with plain `http` scheme. However, it feels like we should use `https` when talking to an https_port. Please test TLS mode with `mgr:command` URLs regardless of whether you decide to use the `https` scheme.